modules/exploits/windows/fileformat/boxoft_wav_to_mp3.rb

```diff
@@ -0,0 +1,62 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
```

Comment:
I know this is copypasta'd ad infinitum, but I think `Msf::Exploit` alone is fine for a file format exploit like this. It's one of the few "omni" exploit types.